jekyll/_cci2/release/configure-release-markers.adoc

```diff
@@ -0,0 +1,46 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Configure release markers
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to configure release markers in your workflow
+:icons: font
+:experimental:
+
+This tutorial shows how to add a "release marker" step to your workflow config. Release markers allow you log all new releases in one place and link back to the CI/CD pipelines that caused them, without needing to install the release agent.
+
+[#prerequisites]
+== Prerequisites
+
+Before adding a release marker step to your workflow, you will need to create a new environment integration in the CircleCI Releases UI.
```

Comment:
As this is a standalone guide for release markers outside the rest of the release features docs, I think this whole bit is step 1 rather than a prerequisite?

Reply:
Good callout. I'll update it.